internal/oci/notation/notation_test.go

```diff
@@ -537,6 +542,61 @@ func TestRepoUrlWithDigest(t *testing.T) {
 	}
 }
 
+func TestVerificationWithProxy(t *testing.T) {
+	g := NewWithT(t)
+
+	registryAddr := testregistry.New(t)
+
+	tarFilePath := path.Join("..", "..", "controller", "testdata", "podinfo", "podinfo-6.1.5.tar")
+	_, err := testregistry.CreatePodinfoImageFromTar(tarFilePath, "6.1.5", registryAddr)
```

Comment:
Differently from `cosign`, this image creation is here because it makes the line https://github.com/fluxcd/source-controller/blob/e1ff038fca2eceaa69a9dbe8a2735c2a01ba0d98/internal/oci/notation/notation.go#L232 not return an error and allows the line https://github.com/fluxcd/source-controller/blob/e1ff038fca2eceaa69a9dbe8a2735c2a01ba0d98/internal/oci/notation/notation.go#L242 to execute. The latter is the line we actually want to test.